veracruz-mcu-client/claps_to_header.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python3
```

Comment:
Can move to veracruz-mcu-client/demo (see above).

The claps.wav binary is a valid concern, it's come up before (#171, we now have an image in the repo), and will probably continue to come up as we expand the number of demos we have.

Should we create a `veracruz-data` repo to capture these? It could be included as a submodule to still allow offline compilation without bloating the size of Veracruz's core git history.

Reply:
What is the license on the `claps.wav` file?  Do we need another `LICENSE` file or is it licensed under one of our existing licenses (e.g. the CC_BY_SA 2.0 license used for the photo)?

Reply:
It's literally a recording of myself clapping at various distances from a microphone, so we own the copyright. It should be fine to release under the existing Veracruz license.